src/pages/workspace/accounting/qbo/advanced/QuickbooksAccountSelectPage.tsx

```diff
@@ -76,28 +76,26 @@ function QuickbooksAccountSelectPage({policy}: WithPolicyConnectionsProps) {
     );
 
     return (
-        <AccessOrNotFoundWrapper
-            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+        <SelectionScreen
             policyID={policyID}
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
             featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
-        >
-            <ScreenWrapper
-                includeSafeAreaPaddingBottom={false}
-                testID={QuickbooksAccountSelectPage.displayName}
-            >
-                <HeaderWithBackButton title={translate('workspace.qbo.advancedConfig.qboBillPaymentAccount')} />
-
-                <SelectionList
-                    sections={qboOnlineSelectorOptions.length ? [{data: qboOnlineSelectorOptions}] : []}
-                    ListItem={RadioListItem}
-                    headerContent={listHeaderComponent}
-                    onSelectRow={saveSelection}
-                    shouldDebounceRowSelect
-                    initiallyFocusedOptionKey={initiallyFocusedOptionKey}
-                    listEmptyContent={listEmptyContent}
-                />
-            </ScreenWrapper>
-        </AccessOrNotFoundWrapper>
+            displayName={QuickbooksAccountSelectPage.displayName}
+            sections={qboOnlineSelectorOptions.length ? [{data: qboOnlineSelectorOptions}] : []}
+            listItem={RadioListItem}
+            headerContent={listHeaderComponent}
+            onSelectRow={saveSelection}
+            shouldDebounceRowSelect
+            initiallyFocusedOptionKey={initiallyFocusedOptionKey}
+            listEmptyContent={listEmptyContent}
+            title="workspace.qbo.advancedConfig.qboBillPaymentAccount"
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.QBO}
+            onBackButtonPress={() => Navigation.goBack()}
+            pendingAction={settingsPendingAction([CONST.QUICKBOOKS_CONFIG.REIMBURSEMENT_ACCOUNT_ID], qboConfig?.pendingFields)}
+            errors={ErrorUtils.getLatestErrorField(qboConfig, CONST.QUICKBOOKS_CONFIG.REIMBURSEMENT_ACCOUNT_ID)}
+            errorRowStyles={[styles.ph5, styles.mv2]}
```

Comment:
Hey 👋 @war-in @zfurtak , I wanted to check if we really need to include the fallback route here. we encountered an issue https://github.com/Expensify/App/issues/51696 where adding the fallback caused a navigation loop, so I’m wondering if removing it and using `Navigation.goBack` instead would cause any known issues

Reply:
Hello @jayeshmangwani, we didn't encountered any problem with the bare `goBack` here, adding fallback route here was more in case of possible problems with lack of it . So I think it's okay to remove it if it solves the issue. 😊